coordinator/src/test/java/org/apache/uniffle/coordinator/SimpleClusterManagerTest.java

```diff
@@ -451,6 +451,8 @@ public void updateExcludeNodesTest() throws Exception {
           remainNodes, availableNodes.stream().map(ServerNode::getId).collect(Collectors.toSet()));
 
       final Set<String> nodes2 = Sets.newHashSet("node3-1999", "node4-1999");
+      // Make sure last modification time change
+      Thread.sleep(1000);
```

Comment:
Could use Awaitility.await() instead of sleep? We should wait for a condition instead of a fixed interval.